fe/fe-core/src/main/java/org/apache/doris/alter/Alter.java

```diff
@@ -260,17 +251,11 @@ private void processModifyColumnComment(Database db, OlapTable tbl, List<AlterCl
         }
     }
 
-    public void replayModifyComment(ModifyCommentOperationLog operation) {
+    public void replayModifyComment(ModifyCommentOperationLog operation) throws MetaNotFoundException {
```

Comment:
It is considerable. But "replay" throwing any exception is an extremely big risk, which will cause all FE crush and cannot recover. These `MetaNotFoundException` are mostly thrown by getDb and getTable, due to the lock inconsistence that makes editlogs out of order. Semantically, throwing this exception means some metadata the editlog want to edit on is missing during replay, which makes this replay unable to continue. Just like getDb/Table, This kind of inconsistence cannot recover anyway, and these editlogs are theoretically only affect lost metadata.
I prefer to use `MetaNotFoundException` that indicate metadata has lost rather than other exception which may cause confuse. The inconsistence can be checked from warning log.

Reply:
@ccoffline   how about check db or table state after getting db or table lock , so that the edit log will always keep in order? we don't need some much check code?

Reply:
@caiconghui we don't have any promise that the edit logs are in order. These check code is to prevent the worst from happening.
Edit logs that out of order may cause meta inconsistent, which has to be fixes sooner or later. We are exploring ways to ensure consistency and minimize the cost. Until then, we have to check all NPE or let all the FE to crash.

Reply:
@ccoffline  so we only need to simply check null for getdb and get table in replay operation and just simply skip the following operation?

Reply:
is it better to split this PR into two PRs ? one for code refactor and another for db and table null check, I think it is easy to review and  not-error-prone when PR is not so big? @ccoffline